packages/expo-updates/src/Updates.ts

```diff
@@ -169,6 +170,41 @@ export async function checkForUpdateAsync(): Promise<UpdateCheckResult> {
   return result;
 }
 
+/**
+* Retrieves the most recent expo-updates log entries from the client.
+ *
+ * @param maxAge Sets the max age of retrieved log entries in seconds. Default to 3600 seconds (1 hour).
```

Comment:
similar to the previous comment, in my experience it is more natural for JS developers to use milliseconds, and I think we use milliseconds more commonly than seconds in other Expo APIs (certainly in expo-updates, with `fallbackToCacheTimeout`), so would advocate for that here as well

Reply:
Ok

Reply:
Done